java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -23,6 +23,7 @@
 import java.time.ZoneOffset;
 import java.time.chrono.ChronoLocalDate;
 import java.time.format.DateTimeFormatter;
+import org.apache.commons.lang.ArrayUtils;
```

Comment:
Although ORC-734 removed `commons-lang` usage from Apache ORC in favor of `lang3`, this seems to bring it back. I'll make a PR to remove it.

Reply:
Here is the PR.
- https://github.com/apache/orc/pull/811